api/database/factories/ApplicantFilterFactory.php

```diff
@@ -86,8 +86,10 @@ public function withRelationships(bool $sparse = false)
             )->get();
             $filter->pools()->saveMany($pools);
             $filter->qualifiedClassifications()->saveMany($pools->flatMap(fn ($pool) => $pool->classifications));
-            $streams = $pools->flatMap(fn ($pool) => $pool->stream);
-            $filter->qualified_streams = $streams;
+            $stream = (empty($pools) || count($pools) === 0) ? $this->faker->randomElements(
+                ApiEnums::poolStreams(),
+            ) : [$pools[0]->stream];
+            $filter->qualified_streams = $stream;
```

Comment:
Filters did not appear to have streams seeded, so I tweaked it while trying to maintain the spirit of the snippet.

Reply:
Yeah, qualified streams are what is attached to an applicant filter

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/2a0c6c00-df4c-4be1-ac19-67279b3d90d6)

Reply:
So you mean you cannot do the below? Is there something being cached at play here?

![gif](http://g.recordit.co/P2obaHQe90.gif)

Reply:
Nope 🤷‍♀️ I see the streams are present on the reuqests but for some reason they do not appear when filtering by that stream 👎 All of the other filters work great though 👍 

https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/e16c0972-f6cb-4bd7-a738-63fc7ac40e68

Reply:
@petertgiles 
We need a referee, does filtering requests by stream work for you? Mind you I tweaked the seeder so you'll need to re-seed requests first

Reply:
Ding ding ding!  I rule in favour of Vachan.

https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/4147f671-38ba-4eb0-927e-8d44022601a7

Reply:
Someone will need to hit that Approve in that case, who will be the lucky person?